lib/guardian.ex

```diff
@@ -366,10 +410,16 @@ defmodule Guardian do
     |> Map.delete("perms")
   end
 
-  defp set_ttl(claims) do
-    claims
-    |> Guardian.Claims.ttl
-    |> Map.delete("ttl")
+  defp set_ttl(claims, type) do
+    if type === "refresh" do
```

Comment:
yes, but this ensures that we can set a default living time for the refresh tokens